SocialCareCaseViewerApi/V1/Factories/EntityFactory.cs

```diff
@@ -180,7 +180,7 @@ public static CaseNotesDocument ToEntity(this CreateCaseNoteRequest request)
         {
             GenericCaseNote note = new GenericCaseNote()
             {
-                Timestamp = DateTime.Now.ToString("dd/MM/yyy hh:mm"),
+                Timestamp = DateTime.UtcNow.ToString("dd/MM/yyyy H:mm:ss"),
```

Comment:
Should a test also need to change to reflect this?

Reply:
This is one of the tech debt areas because we don't have any test coverage for this. I'll add some now so it's not missing completely

Reply:
@chrishutchinson I've added two tests that cover the object conversion as well as the date time format for timestamp. It's not perfect since checking the date time after it's been converted to string is bit of a challenge in terms of checking whether it was created using 24h format or not. These tests should protect against the main format changes at least.

Reply:
Nice one @LBHTKarki! That makes, and looks good, approved ✅